test/framework/vmoperator/vmoperator.go

```diff
@@ -470,97 +440,6 @@ func ReconcileDependencies(ctx context.Context, c client.Client, dependenciesCon
 		contentLibraryID = id
 	}
 
-	contentSource := &vmoprv1.ContentSource{
```

Comment:
Simila question here

Also let's update the comment in l.397++

Reply:
It got superseded are reconciled by a different controller: github.com/vmware-tanzu/image-registry-operator-api/api/v1alpha1.

It is just enough to have the `VirtualMachineImage` + properly setting status instead of having the additional dependency (go pkg) and creating the resources reconciled by a not running controller.

These CRs are not needed when we just need to have the VirtualMachineImage where we already patch status accordingly.